### PR TITLE
Adds the navbar for the frontpage.

### PR DESCRIPTION
**Pull request by abundis28**
A problem is that in this branch there is no hobbies.html included, but I did include the link in the navbar. That will probably only make sense once this PR is merged with master along with the hobbies-feature PR.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -3,10 +3,30 @@
...
     <script src="script.js"></script>
+    <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js" integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js" integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo" crossorigin="anonymous"></script>
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js" integrity="sha384-OgVRvuATP1z7JjHLkuOU7Xw704+h835Lr+6QL9UvYjZE3Ipu6Tp75j7Bh/kR0JKI" crossorigin="anonymous"></script>
   </head>
   <body>
+    <div id="barTop">
+      <nav class="navbar navbar-expand-lg navbar-dark" style="background-color: #004047">
+        <a class="navbar-brand" href="#">Andrés's Portfolio</a>
+        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
```
Comment (timothy-parsons) [inline portfolio/src/main/webapp/index.html]: Consider wrapping this line - https://google.github.io/styleguide/htmlcssguide.html#HTML_Line-Wrapping

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -26,12 +26,12 @@ body {
 h1 {
   color: white;
 }
-
+
```
Comment (timothy-parsons) [inline portfolio/src/main/webapp/style.css]: Watch you aren't adding errant spaces!

**Comment by timothy-parsons** [comment]
> A problem is that in this branch there is no hobbies.html included, but I did include the link in the navbar. That will probably only make sense once this PR is merged with master along with the hobbies-feature PR.

I would say your intuition is correct, and I'll repeat what I said in that PR here - ideally you never are pushing a broken state to master. That might mean holding off on adding the hobbies.html link until the PR that introduces hobbies.html.

**Review by timothy-parsons — CHANGES_REQUESTED** [review]
Looks mostly good! Just a few small comments!

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (abundis28) [reply portfolio/src/main/webapp/index.html]: Done!

File: portfolio/src/main/webapp/style.css (hunk 2 above)
Comment (abundis28) [reply portfolio/src/main/webapp/style.css]: Ack, sorry about that!

**Review by timothy-parsons — APPROVED** [review]
LGTM, good job!

**Pull request merged**